site/content/en/docs/api_sdk/cli/_index.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: 'Command line interface (CLI)'
 linkTitle: 'CLI'
-weight: 29
-description: 'Guide to working with CVAT tasks in the command line interface. This section on [GitHub](https://github.com/cvat-ai/cvat/tree/develop/cvat-cli).'
+weight: 4
+description: ''
```

Comment:
```suggestion
description: 'Python command line interface to interact with a CVAT instance'
```

Reply:
This description is only displayed as an extra line under the title. I'm not sure this description adds anything to the title